core/templates/dev/head/pages/exploration_editor/EditorServicesSpec.js

```diff
@@ -831,33 +831,34 @@ describe('New state template service', function() {
 
     it('should make a new state template given a state name', function() {
       expect(JSON.parse(JSON.stringify(
-        nsts.getNewStateTemplate(NEW_STATE_NAME)))).toEqual({
-        name: 'new state name',
-        content: [{
-          type: 'text',
-          value: ''
-        }],
-        interaction: {
-          answer_groups: [],
-          confirmed_unclassified_answers: [],
-          customization_args: {
-            rows: {
-              value: 1
+          nsts.getNewStateTemplate(NEW_STATE_NAME)
```

Comment:
Actually, let's do this in the next PR, rather than go through another round of Travis checks. Let's merge this!